experimental/web_dashboard/lib/src/pages/dashboard.dart

```diff
@@ -0,0 +1,66 @@
+// Copyright 2020, the Flutter project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'package:web_dashboard/src/api/api.dart';
```

Comment:
This is referenced in an earlier comment as well, but it looks like some imports of internal files are `package:`-style and some are direct paths. Whichever one you prefer is fine, but I'd pick a type and stick with it for readability.

Reply:
Changed the imports to relative ones: https://github.com/flutter/samples/pull/421/commits/4e47b92109adce3efaf29cff826ee254327e2dd9